code/game/gamemodes/nuclear/nuclear.dm

```diff
@@ -194,6 +194,7 @@ proc/issyndicate(mob/living/M as mob)
 	M.s_tone = skin_tone
 	M.h_style = hair_style
 	M.f_style = facial_hair_style
+	M.body_accessory = null

```

Comment:
Lol...just....this is hilarious--in some ways I wished it remained in as an ultra-low probability thing (like female ops with facial hair).

That said, definitely for the best, hahaha =p